tools/commands/build.py

```diff
@@ -66,37 +86,100 @@ def make(self):
         logger.info("building & installing {}: {}".format(self.name, " ".join(cmd)))
         subprocess.check_call(cmd, env=self.env)
 
+class acbuilder(builder):
+    def __init__(self, name, modules_dir, build_dir, install_dir, make_verbose=False):
+        self.make_verbose = make_verbose
+        super(acbuilder, self).__init__(name, modules_dir, build_dir, install_dir)
+
+    def clean(self):
+        if os.path.exists(self.build_path):
+            os.system("make -C {} uninstall".format(self.build_path))
+        super(acbuilder, self).clean()
+
+    def prepare(self):
+        os.chdir(self.src_path)
+        cmd = ["./build-tools/autogen.sh", "-f"]
+        logger.info("preparing {}: {}".format(self.name, " ".join(cmd)))
+        subprocess.check_call(cmd, env=self.env)
```

Comment:
Consider adding stamping like done in cmakebuilder to skip steps if already done:
```
  if os.path.exists("{}/.prepared".format(self.build_path)):
            logger.info("{} already prepared, skip {}".format(self.build_path, self.name))
...
...
open("{}/.prepared".format(self.build_path), 'a').close()
```

Reply:
but we can certainly do this later, no point in keeping this waiting